source/class/cv/util/ConfigLoader.js

```diff
@@ -104,21 +104,35 @@ qx.Class.define('cv.util.ConfigLoader', {
           if (cv.Config.libraryCheck && xmlLibVersion < cv.Version.LIBRARY_VERSION) {
             this.configError('libraryerror');
           } else {
+            cv.Config.server = {};
             let backendName = '';
             if (req.getResponseHeader('X-CometVisu-Backend-Name')) {
               backendName = req.getResponseHeader('X-CometVisu-Backend-Name');
             }
-            if (req.getResponseHeader('X-CometVisu-Backend-LoginUrl')) {
-              cv.Config.backendUrl = req.getResponseHeader('X-CometVisu-Backend-LoginUrl');
-              if (!cv.Config.backendUrl.endsWith('/')) {
-                cv.Config.backendUrl += '/';
+            if (req.getResponseHeader('X-CometVisu-Backend-KNXD-Url')) {
+              cv.Config.server.backendKnxdUrl = req.getResponseHeader('X-CometVisu-Backend-KNXD-Url');
+              if (backendName === '') {
+                backendName = 'knxd';
               }
-              if (!backendName && cv.Config.backendUrl.startsWith('/rest/')) {
+            }
+            if (req.getResponseHeader('X-CometVisu-Backend-MQTT-Url')) {
+              cv.Config.server.backendMQTTUrl = req.getResponseHeader('X-CometVisu-Backend-MQTT-Url');
+              if (backendName === '') {
+                backendName = 'mqtt';
+              }
+            }
+            if (req.getResponseHeader('X-CometVisu-Backend-OpenHAB-Url')) {
+              cv.Config.server.backendOpenHABUrl = req.getResponseHeader('X-CometVisu-Backend-OpenHAB-Url');
+              if (backendName === '') {
                 backendName = 'openhab';
               }
             }
+            // legacy support, reproduce old behaviour
+            if (!req.getResponseHeader('X-CometVisu-Backend-Name') && cv.Config.backendOpenHABUrl.startsWith('/rest/')) {
```

Comment:
This is obsolete now, because the name is already set when we habe a backendOpenHABUrl